documentation/schema/Syriacamsdesc.odd

```diff
@@ -4634,8 +4900,251 @@
                             </p>
                         </div>-->
                     <div xml:id="decodesc">
-                        <head>Decoration Description [UNDER REVISION]</head>
-                        <p>Further documentation will be added here.</p>
+                        <head>Decoration Description</head>
+                        <p><title>SMBL</title> has used <gi>decoNote</gi> elements within the <gi>decoDesc</gi>
+                           to record information about decorative features in a manuscript. Each <gi>decoNote</gi> element
+                           may have a <gi>locus</gi> element, which records the folio location(s) on which the decorative
+                           elements appear using <att>from</att>, and optionally <att>to</att>, attributes. A prose
+                           text node records Wright's description of the decorative feature, with the exception of <val>rubrication</val>
+                           where a text node was omitted if Wright does not provide additional details.</p>
+                        <p>Each <gi>decoNote</gi> element has a <att>type</att> attribute.
+                            <title>SMBL</title> uses a subset of the Fihrist vocabulary, with one addition <val>ornamentation</val> which 
```

Comment:
I see that you have properly used "val" here which is right, but the HTML transform does not render it, so I have been using "q". We could change mine or discuss.